src/lib/wallet/GoodWalletClass.js

```diff
@@ -198,6 +198,10 @@ export class GoodWallet {
     return this.ready
   }
 
+  getSignUpBonusAddress() {
```

Comment:
we already have this.SignupContract.
to get address use this.SignupContract.address, no need for this method, use it inline

Reply:
@sirpy there is no declaration and usage of SignUpBonusContract in GoodWalletClass.js file. So I made this fn to get SignUpBonus address depends on networkId which is defined here.